test/errors.go

```diff
@@ -0,0 +1,9 @@
+package gonawintest
+
+// ErrString returns the string representation of an error.
+func ErrString(err error) string {
```

Comment:
Would you name the function `ErrorStringRepresentation` or at least `ErrorString` ?